pandas/core/series.py

```diff
@@ -251,6 +251,38 @@ class Series(base.IndexOpsMixin, generic.NDFrame):
     Note that the Index is first build with the keys from the dictionary.
     After this the Series is reindexed with the given Index values, hence we
     get all NaN as a result.
+
+    Constructing Series from a list with `copy=False`.
+
+    >>> r = [1, 2]
+    >>> ser = pd.Series(r, copy=False)
+    >>> ser.iloc[0] = 999
+    >>> r
+    [1, 2]
+    >>> ser
+    0    999
+    1      2
+    dtype: int64
+
+    Due to input data type the Series has a `copy` of
+    the original data even though `copy=False`, so
+    the data is unchanged.
+
+    Constructing Series from a 1d ndarray with `copy=False`.
+
+    >>> r = np.array([1, 2])
+    >>> ser = pd.Series(r, copy=False)
+    >>> ser.iloc[0] = 999
+    >>> r
+    array([999,   2])
+    >>> ser
+    0    999
+    1      2
+    dtype: int32
```

Comment:
looks like this test is failing in CI:
```
=================================== FAILURES ===================================
_____________________ [doctest] pandas.core.series.Series ______________________
269     the data is unchanged.
270 
271     Constructing Series from a 1d ndarray with `copy=False`.
272 
273     >>> r = np.array([1, 2])
274     >>> ser = pd.Series(r, copy=False)
275     >>> ser.iloc[0] = 999
276     >>> r
277     array([999,   2])
278     >>> ser
Differences (unified diff with -expected +actual):
    @@ -1,3 +1,3 @@
     0    999
     1      2
    -dtype: int32
    +dtype: int64
```

Reply:
Oh got it, I didn't see that. Appreciate the help.